apis/abortion_policy_api.py

```diff
@@ -3,9 +3,15 @@
 import requests
 import os
 import json
+import re
 
 APIKEY = os.environ.get('ABORTION_POLICY_API_KEY')
 HEADERS = {'token': APIKEY}
+TYPE_DEFAULTS = {
+    'str': None,
+    'bool': False,
+    'int': 0,
+    'float': 0.0}
 
 
 def get_states(filepath):
```

Comment:
Since we are never passing another file name here, I think it'd be better to just have it call the file without passing it in.